mocks/main/projects/test/endpoints/POST.js

```diff
@@ -13,7 +13,23 @@ export default function (req, res) {
     }
   }
 
-  if (req.body.endpoint.branch_id === 'test_branch_id') {
+  if (req.body.endpoint.branch_id === 'test_branch_with_fixed_cu') {
```

Comment:
I consider having mock detached from tests bad practice.

Reply:
It works for most of tests, cause they don't need specific conditions to verify the behavior. 
In case where it matters, I agree, let's put mocks next to the test, there is a `mockDir` parameter in testCliCommand helper

Reply:
Can we migrate to `msw` or something similar?

So a test like this
```ts
// projects.test.ts
testCliCommand({
  name: 'create project with default fixed size CU',
  args: ['projects', 'create', '--name', 'test_project_with_fixed_cu', '--cu', '2'],
  expected: {
    snapshot: true,
  },
});
// POST.ts
expect(req.body).toMatchObject(...);
res.send(...);
```
will look like this
```ts
test('create project with default fixed size CU', ({ mock }) => {
  // Mock mechanism should warn if the mock is not used in the end of the test
  // If you forget to mock a network response - the error will be raised as well
  mock<ProjectCreateRequest, ProjectResponse>("projects", ({ request }) => {
    expect(request).toEqual(...);
    return ...; // response
  });
  const result = await testCliCommand(['projects', 'create', '--name', 'test_project_with_fixed_cu', '--cu', '2']);
  expect(result).toHaveInlineSnapshot(...)
});
```

Reply:
How would you approach the reusability of mocks?

Reply:
@duskpoet thoughts?

Reply:
Add helpers to create typical response, with overrides as arguments

Reply:
> Add helpers to create typical response, with overrides as arguments

(This is how we plan to do it internally too)